src/components/AvatarWithImagePicker.tsx

```diff
@@ -330,6 +311,16 @@ function AvatarWithImagePicker({
         [disabled, enablePreview, isUsingDefaultAvatar, onViewPhotoPress, showAvatarCropModal],
     );
 
+    useLayoutEffect(() => {
+        if (!anchorRef.current || !isMenuVisible) {
+            return;
+        }
+
+        calculatePopoverPosition(anchorRef, anchorAlignment).then(({vertical, horizontal, width}) => {
+            setPopoverPosition({vertical: vertical + variables.spacing2, horizontal: horizontal - width + (width - variables.photoUploadPopoverWidth) / 2});
```

Comment:
I see some changes here (additional `- width` and no `+ height` which was added before), was this intentional?